File: FizzBuzz/C#_3/FizzBuzzTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using NUnit.Framework;

namespace Kata
{
    [TestFixture]
    class FizzBuzzTest
    {
        FizzBuzz _sut;

        [OneTimeSetUp]
        public void Inint()
        {
            _sut = new FizzBuzz();
        }

        [Test]
        [TestCase(1)]
        public void Say_NumberIsNotDivisible3Or5_ReturnNumber(int number)
        {
            // Arrange
            // Act
            var result = _sut.Say(number);

            // Assert
            Assert.AreEqual(number.ToString(), result);
        }

        [Test]
        [TestCase(3)]
        public void Say_NumberIsDivisible3_ReturnFizz(int number)
        {
            // Arrange
            // Act
            var result = _sut.Say(number);

            // Assert
            Assert.AreEqual("Fizz", result);
        }

        [Test]
        [TestCase(5)]
        public void Say_NumberIsDivisible5_ReturnBuzz(int number)
        {
            // Arrange
            // Act
            var result = _sut.Say(number);

            // Assert
            Assert.AreEqual("Buzz", result);
        }

        [Test]
        [TestCase(15)]
        [TestCase(30)]
        public void Say_NumberIsDivisible3Or5_ReturnFizzBuzz(int number)
        {
            // Arrange
            // Act
            var result = _sut.Say(number);

            // Assert
            Assert.AreEqual("FizzBuzz", result);
        }
    }
}

File: FizzBuzz/Python_1/FizzBuzzTest.py
import unittest
import FizzBuzz

class FizzBuzzTest(unittest.TestCase):
    def setUp(self):
        self.sut = FizzBuzz.FizzBuzz()

    def test_Say_NumberNotDivisible3Or5_ReturnNumber(self):
        # arrange        
        expect = '1'

        # act
        actual = self.sut.Say(1)

        # assert
        self.assertEqual(expect, actual)

    def test_Say_NumberDivisible3_ReturnFizz(self):
        # arrange
        expect = 'Fizz'

        # act
        actual = self.sut.Say(3)

        # assert
        self.assertEqual(expect, actual)

    def test_Say_NumberDivisible5_ReturnBuzz(self):
        # arrange
        expect = 'Buzz'

        # act
        actual = self.sut.Say(5)

        # assert
        self.assertEqual(expect, actual)

    def test_Say_NumberDivisible3And5_ReturnFizzBuzz(self):
        # arrange
        expect = 'FizzBuzz'

        # act
        actual = self.sut.Say(15)

        # assert
        self.assertEqual(expect, actual)

if __name__ == '__main__':
    unittest.main()